examples/common/src/chariott/registration.rs

```diff
@@ -3,20 +3,20 @@
 
 use std::{env, net::SocketAddr, time::Duration};
 
-use tokio::time::sleep;
-use tonic::transport::Channel;
-use tracing::warn;
-use url::Url;
-
 use chariott_common::{
     config,
     error::{Error, ResultExt},
 };
-
-use crate::chariott::proto::runtime_api::{
-    chariott_service_client::ChariottServiceClient, intent_service_registration::ExecutionLocality,
-    *,
+use chariott_proto::runtime::{
+    chariott_service_client::ChariottServiceClient, intent_registration::Intent,
+    intent_service_registration::ExecutionLocality, AnnounceRequest, IntentRegistration,
+    IntentServiceRegistration, RegisterRequest, RegistrationState,
 };
+use tokio::time::sleep;
```

Comment:
nit: Intentional move?

Reply:
Yes, I applied the same reasoning as in https://github.com/eclipse/chariott/pull/32#discussion_r1022562625.

I would suggest to keep it that way, as it makes it clear where to add new `use` statements.